### PR TITLE
All new way of initing a node.

### PR DESCRIPTION
**Pull request by diasdavid**
For https://github.com/ipfs/docs/issues/22

### CODE REVIEW

**Comment by diasdavid** [comment]
all tests passing, now just have to update all the docs and tests wooo

File: src/core/index.js [hunk 1]
```diff
@@ -9,39 +9,60 @@ const multihash = require('multihashes')
...
-      multiaddr,
-      multihash,
-      CID
+    if (typeof options.init === 'boolean' &&
+        options.init === false) {
+      this._options.init = false
+    }
+
+    if (!(
+        typeof options.start === 'boolean' &&
```
Comment (dignifiedquire) [inline src/core/index.js]: strange line break

File: README.md [hunk 2]
```diff
@@ -196,46 +196,54 @@ The HTTP-API exposed by the js-ipfs daemon follows the [`http-api-spec`](https:/
...
+
+node.on('start', () => {
+  // Your now is ready to use \o/
+
+  // stopping a node
+  node.stop(() => {
+    // node is now 'offline'
+  })
+})
+```
```
Comment (diasdavid) [inline README.md]: This has to make people happier :)

File: README.md [hunk 3]
```diff
@@ -196,46 +196,54 @@ The HTTP-API exposed by the js-ipfs daemon follows the [`http-api-spec`](https:/
...
+  // start: false,
+  EXPERIMENTAL: { // enable experimental features
+    pubsub: true
+  },
+  config: { // overload the default config
+    Addresses: {
+      Swarm: [
+        '/ip4/127.0.0.1/tcp/1337'
+      ]
+    }
```
Comment (diasdavid) [inline README.md]: No more get/set config madness :)

**Comment by diasdavid** [comment]
Can I get some review now? :)

**Pull request merged**